packages/react-components/react-toast/src/components/Toast.tsx

```diff
@@ -99,7 +99,7 @@ export const Toast: React.FC<ToastProps & { visible: boolean }> = props => {
     <Transition in={visible} unmountOnExit mountOnEnter timeout={500} onExited={remove} nodeRef={toastRef}>
       <div ref={toastRef} className={mergeClasses(styles.toast, visible && styles.slide, !visible && styles.fadeOut)}>
         {children}
-        <Timer onTimeout={close} timeout={timeout ?? -1} running={running} />
+        <Timer key={updateId} onTimeout={close} timeout={timeout ?? -1} running={running} />
```

Comment:
As it's DOM specific, is not a better to trigger a reflow instead https://stackoverflow.com/a/45036752/6488546?

Reply:
IMO it's probably more verbose since you'd ref to the actual HTMLElement. It wouldn't  remove the need for the `updateId` because it's still necessary to know when to trigger the overflow